pkg/registry/core/persistentvolume/storage/storage_test.go

```diff
@@ -169,7 +169,7 @@ func TestUpdateStatus(t *testing.T) {
 	storage, statusStorage, server := newStorage(t)
 	defer server.Terminate(t)
 	defer storage.Store.DestroyFunc()
-	ctx := genericapirequest.NewContext()
+	ctx := genericapirequest.NewDefaultContext()
```

Comment:
```suggestion
	ctx := genericapirequest.WithNamespace(genericapirequest.NewContext(), metav1.NamespaceNone)
```
From https://github.com/kubernetes/kubernetes/pull/108068#discussion_r804682973
(I hadn't pushed suggested changes on account of https://github.com/kubernetes/kubernetes/pull/108068#discussion_r808130936

Reply:
ah, right... I squashed the comments into your commit, and split out the test bit that *has* to be part of this PR because it relies on the create/update handler changes

I think the current `Update tests to have namespace set in req context` commit in this PR could actually pass tests and merge to master ahead of this PR if you want to pull it back out... sorry for the churn :)

Reply:
Nope, this looks good! Thanks!
The only thing I can think of is my commit message which contains details pre-your-review, you might have to one last rebase to change that 😬